src/font/fontconfig.c

```diff
@@ -28,6 +28,8 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include "config.h"
+
```

Comment:
I forgot to add cofnig.h to fontconfig.c in https://github.com/lc-soft/LCUI/commit/f9ecdb7fcfcbceaf4667b05f28ad49d9b983db5d

Reply:
I see.